src/components/SiteHeader/SiteHeader.tsx

```diff
@@ -5,56 +5,55 @@ import styles from './SiteHeader.module.css';
 import whiteLogo from '../../images/dk-white-logo.png';
 
 const SiteHeader: React.FunctionComponent = () => {
-
   const history: any = useHistory();
-  const [input, setInput] = useState("");
+  const [input, setInput] = useState('');
 
   function handleEnter(e: any): void {
     if (e.key === 'Enter') {
-      history.push('/search', {searchInput: input})
+      history.push('/search', { searchInput: input });
     }
   }
 
   return (
     <>
-      <div className={styles["site-header"]}>
-        <div className={styles["site-header-top"]}>
-          <div className={styles["digital-kampung-div"]}>
+      <div className={styles['site-header']}>
+        <div className={styles['site-header-top']}>
+          <div className={styles['digital-kampung-div']}>
             <Link
-              style={{textDecoration: 'none', color: "black"}}
+              style={{ textDecoration: 'none', color: 'black' }}
               to={{
-                pathname: "/"
+                pathname: '/',
               }}
             >
-              <img className={styles["digital-kampung-logo"]} src={whiteLogo} alt=""></img>
+              <img className={styles['digital-kampung-logo']} src={whiteLogo} alt=""></img>
             </Link>
           </div>
-          <div className={styles["header-button-group"]}>
-            <Button className={styles["header-button"]}>Kampung Centre</Button>
-            <Button className={styles["header-button"]}>Sign up</Button>
-            <Button className={styles["header-button-primary"]}>Log in</Button>
+          <div className={styles['header-button-group']}>
+            <Button className={styles['header-button']}>Kampung Centre</Button>
```

Comment:
It is confirmed that we use "Kampung" instead of "Kampong" right?

Reply:
Yes